Client/Extensions/UIFont+Extension.swift

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit.UIFont
+
+extension UIFont {
```

Comment:
Wondering if this could even go under `Common` target in BrowserKit?

Reply:
I would argue against that, simply because this is something Swift has implemented, and we should only be using this for a very minor set of cases. 
I've created [this ticket to address this](https://mozilla-hub.atlassian.net/browse/FXIOS-6364).